GoogleForCreators/web-stories-wp
# Templates: Update Some Images to Shapes

## Summary
This PR just goes through some templates that were using images for icons we can use shapes for and updates those images to shapes.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
social icons in DIY & travel and star icons in Entertainment should now all be shapes instead of images

## Testing Instructions
Go to explore templates
-> Create from template for DIY
-> Go to the page with the social icons and see that you can change their color
-> Verify that there aren't any regressions in the template (just quick scroll through all pages and see that no images are missing or anything glaring like that. Should be fine tho)

Repeat these steps for the social icons in the travel template and the star Icons in the entertainment template.

---



Partially addresses #5873
